### PR TITLE
Added a new two-tile motel

### PR DESCRIPTION
**Pull request by Night-Pryanik**
#### Summary
```SUMMARY: Content "Adds a new two-tile motel."```

#### Purpose of change
Variety.

#### Describe the solution
Added a new two-tile location based on a location from some popular video game. 
It isn't a very loot-heavy or very interesting to visit, but was designed as more of a tribute to the game (I hope it is recognizable to which one).

Incidentally linted the `motel.json`.

For some unknown reason I didn't manage to spawn any monster on a location despite the syntax seems to be right.

#### Additional context
изображение

### CODE REVIEW

**Comment by Ilysen** [comment]
If I'm recognizing the shape of this building right, perhaps you should make one of the rooms spawn some meth!

**Comment by Night-Pryanik** [comment]
No, it's not the mobile meth lab, it's RV.
<details> 
  <summary>The sign on the northern side reads (spoilers below)</summary>
Travelier Motel 
</details>

**Comment by Ilysen** [comment]
It also shares a very similar shape to one of the missions in Payday 2, where you cook meth in one of the rooms. I guess I misunderstood the reference!

File: data/json/mapgen/motel.json [hunk 1]
```diff
@@ -1363,10 +1360,183 @@
...
+        "t": { "item": "book_motel", "chance": 100 },
+        "Y": { "item": "coat_rack", "chance": 15 },
+        ":": { "item": "dresser", "chance": 10 },
+        "b": { "item": "bed", "chance": 40 },
+        ",": { "item": "trash_forest", "chance": 1 }
+      },
+      "place_items": [ { "item": "maps", "x": 4, "y": 3, "chance": 70 } ],
+      "place_item": [ { "item": "wheel", "x": 11, "y": 8, "chance": 80 } ],
+      "toilets": { "&": {  } },
+      "place_signs": [ { "signage": "Travelier MOTEL", "x": 23, "y": 1 } ]
```
Comment (ZhilkinSerg) [inline data/json/mapgen/motel.json]:
```suggestion
      "place_signs": [ { "signage": "Travelier MOTEL", "x": 23, "y": 1 } ],
```

File: data/json/mapgen/motel.json [hunk 2]
```diff
@@ -1363,10 +1360,183 @@
...
+    },
```
Comment (ZhilkinSerg) [inline data/json/mapgen/motel.json]:
```suggestion
```

File: data/json/mapgen/motel.json [hunk 3]
```diff
@@ -1363,10 +1360,183 @@
...
+    "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 7, 22 ], "y": [ 5, 16 ] } ]
```
Comment (ZhilkinSerg) [inline data/json/mapgen/motel.json]: I guesss `place_monster` has invalid parent node - it should be under `object`.

```suggestion
      "place_monsters": [ { "monster": "GROUP_ZOMBIE", "x": [ 7, 22 ], "y": [ 5, 16 ] } ]
    },
```

**Comment by LovamKicsiGazsii** [comment]
```t_floor``` should be ```t_pavement``` here. 

tile

**Comment by Night-Pryanik** [comment]
> t_floor should be t_pavement here.

Done.

**Comment by Night-Pryanik** [comment]
Jenkins rebuild

**Pull request merged**